modules/next_stage/test/TestPlan.md

```diff
@@ -1,33 +1,34 @@
+# Next Stage - Test Plan
 1. Create a new candidate and create new timepoint for them.
    [Manual Testing]
-2. Click "Start Visit Stage" on left of instrument_list ensure that page
+2. Click "Start Visit Stage" on left of `instrument_list` ensure that page
    renders.
    [Manual Testing]
-3. Ensure that stage can only be started with the data_entry permission
+3. Ensure that stage can only be started with the `data_entry` permission
    and for candidates at the user's site.
    [Automation Testing]
 4. Ensure that you get an error if dates do not match.
    [Automation Testing]
-5. Ensure that you get an error if scan_done missing
+5. Ensure that you get an error if `scan_done` missing
    [Manual Testing]
 6. Create a timepoint where the age (date entered minus DoB) falls within the
-   AgeMin and AgeMax for a test_battery. Ensure that those instruments are
+   AgeMin and AgeMax for a `test_battery`. Ensure that those instruments are
```

Comment:
AgeMin and AgeMax for a `test_battery`. Those test_battery needs to have a matching subprojectID and a NULL Visit_label. Ensure that those instruments are